### PR TITLE
Lexer can now correctly tokenise operators with 3+ characters

### PR DESCRIPTION
**Pull request by RobinLockyer**
Replaced readMultiCharOperator() with readOperator() which can read arbitary length operators Fixed bug where the default GojoToken.String() was being called instead of the one specified in lexer/token.go Fixed above bug for GojoTokenType.String() and GojoTokenType.StringInline() also Moved "?" token to TokenOperators map
Gave lexer tests meaningful names
Added new lexer test: "MultiCharacterOperators"
Added helper functions to make writing lexer tests easier. New functions are:

- 	NewToken(tokenTypeStr string, text ...string) GojoToken
- 	NewNumber(number string) GojoToken
- 	NewID(id string) GojoToken
- 	NewString(str string)

Rewrote all existing lexer tests using said helper functions

### CODE REVIEW

File: lexer/lexer.go [hunk 1]
```diff
@@ -55,27 +56,27 @@ func (l *Lexer) NextToken() GojoToken {
 			tok = l.NewToken(TokenOperators["/"], string(l.ch))
 		}
 	case '=':
```
Comment (GitPaulo) [inline lexer/lexer.go]: can you inline all these operators like so:
```go
case '+', '-' ...
   tok = l.readOperator()
```

File: tests/data/lexer/Functions.js [hunk 2]
```diff
@@ -2,4 +2,3 @@ function add(a, b) {
 	return a + b;
 }
 var result = add(5, 10);
-`,
```
Comment (GitPaulo) [inline tests/data/lexer/Functions.js:5]: EOF missing

**Review by GitPaulo — CHANGES_REQUESTED** [review]
nice!

File: lexer/lexer.go (hunk 1 above)
Comment (RobinLockyer) [reply lexer/lexer.go]: Fixed in latest commit

File: tests/data/lexer/Functions.js (hunk 2 above)
Comment (RobinLockyer) [reply tests/data/lexer/Functions.js:5]: Fixed in latest commit

**Pull request merged**